lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -98,5 +98,10 @@ public class TopologyRootTaskId : Name<string>
         public class TopologyChildTaskIds : Name<ISet<string>>
         {
         }
+
+        [NamedParameter("Type of the message")]
+        public class MessageType : Name<string>
```

Comment:
The reason is I cannot get generic argument type from the Configuration. 

For example, I can bind the following to the same Configuration at driver side. 
BindImplementation(GenericType<IMyOperator<int>>.Class, GenericType<MyOperatorImpl<int>>.Class)          
                .BindImplementation(GenericType<IMyOperator<string>>.Class, GenericType<MyOperatorImpl<string>>.Class)

At task side, to get instance from the interface IMyOperator<T> using Tang, I must know T. e.g. either call 
injector.GetInstance(IMyOperator<int>) or call injector.GetInstance(IMyOperator<string>). They are different interface. 
I have no way to derive the T from the Configuration. 

You can find examples from TestInjectionWithT()